src/command/output.rs

```diff
@@ -447,6 +448,7 @@ impl RoverOutput {
             }
             RoverOutput::CheckResponse(check_response) => check_response.get_json(),
             RoverOutput::AsyncCheckResponse(check_response) => check_response.get_json(),
+            RoverOutput::LintResponse(lint_response) => lint_response.get_json(),
```

Comment:
further down in this file there are some tests for the json output - these tests are there to make sure that we keep the interface stable for anybody consuming rover output programatically

Reply:
Unit test has been added